OKR/JS_CORE.md

```diff
@@ -2,6 +2,28 @@
 
 We frame our ongoing work using a process based on quarterly Objectives and Key Results (OKRs). Objectives reflect outcomes that are challenging, but realistic. Results are tangible and measurable.
 
+## 2019 Q1
+
+### A complete implementation of IPFS in JavaScript
+
+* p0 - Base32 encoded version 1 CIDs are the default for new content added to IPFS [@alanshaw](https://github.com/alanshaw)
+* p0 - IPNS has a streaming API [@vasco-santos](https://github.com/vasco-santos)
```

Comment:
:+1: as this would enable https://github.com/ipfs-shipyard/ipfs-companion/issues/643 with js-ipfs :)